docs/version0.18.md

```diff
@@ -40,6 +40,7 @@ The following new features and notable changes since v 0.17.0 are included in th
 - [Improved support for pause-less garbage collection](#improved-support-for-pause-less-garbage-collection)
 - [`-Xgc:noConcurrentScavenge` option](#-xgcnoconcurrentscavenge-option)
 - [Support for OpenJDK HotSpot options](#support-for-openjdk-hotspot-options)
+- [`-Djava.lang.string.substring.nocopy` option](#-djavalangstringsubstringnocopy)
```

Comment:
This link at the top of the version.18 topic isn't working. I think it should be:
`(#-djavalangstringsubstringnocopy-option)`

Reply:
Fixed